src/components/Combobox/__tests__/combobox-interactions.js

```diff
@@ -0,0 +1,92 @@
+import { expect } from "@storybook/jest";
+import {
+  ELEMENT_TYPES,
+  getByLabelText,
+  getByTestId,
+  getByText,
+  clickElement,
+  resetFocus,
+  typeText,
+  interactionSuite,
+  pressNavigationKey
+} from "../../../__tests__/interactions-helper";
+import { queryByText } from "@storybook/testing-library";
+import { getTestId, NAVIGATIONS_COMMANDS } from "utils/test-utils";
+
+async function getSearchElement(canvas) {
```

Comment:
I think the function name needs to be updated (you don't get just the search element). WDYT about `getComponentElements`?